content/api/editor-support.md

```diff
@@ -7,17 +7,27 @@ Installing our code locally will allow your editor to extract information from o
 Installing these definitions is optional, however will be incredibly useful if your editor supports this. All editors in our [recommended list](/tutorials/editors/) besides IDLE will support it.
 
 ## Installation
-Installing our API, the code you interact with, can be installed using `pip`, Python's built-in package manager. `pip` should be installed on your machine, if it's not, you'll need to look up how to install it.
+Installing our API, the code you interact with, can be installed using `pip`, Python's built-in package manager. `pip` should be installed on your machine, if it's not, you'll need to look up how to install it. 
 
 ```bash
 pip install git+https://github.com/sourcebots/robot-api
 ```
+And on windows:
```

Comment:
I'm not sure what the cause of difference is, but I've always been able to just `pip` directly on windows.

Reply:
It's possible this addition isnt necessary, but when I tried yesterday on @BitBait 's windows machine, `pip` wasn't available in the path. I'll investigate on my windows machine(s) and see if I can find a cause

Reply:
I can use `pip` on Windows without explicitly editing the path, the installer has a tick box (ticked by default) for auto adding installation to PATH